src/components/interactive/Avatar.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import baseElement from '../../util/baseElement';
+
+export const VALID_BG_COLORS = [
+  'white',
+  'haze',
+  'lightGray',
+  'orange',
+  'charcoal',
+  'navy',
+  'aqua',
+  'gray',
+  'red',
+];
+
+const INVERTED_BG_COLORS = ['gray', 'charcoal', 'navy', 'orange'];
+
+const grabInitials = (str) =>
+  str
+    .split(' ')
+    .splice(0, 2)
+    .reduce((prev, curr) => prev + curr.charAt(0), '');
+
+const Avatar = ({ bgColor, imgUrl, size, name, Link, ...rest }) => {
+  const Element = baseElement({ href: rest.href, as: Link });
+  return (
+    <Element
+      {...rest}
+      title={name}
+      className={cx('fdsAvatar', `bgColor--${bgColor}`, `fdsAvatar--${size}`, {
+        'color--white': INVERTED_BG_COLORS.includes(bgColor),
+      })}
+    >
+      {imgUrl && (
+        <span className="fdsAvatar-img" style={{ backgroundImage: `url(${imgUrl})` }} />
+      )}
+      {name && grabInitials(name)}
+    </Element>
+  );
+};
+
+Avatar.defaultProps = {
+  bgColor: 'gray',
+  size: 'm',
+};
+
+Avatar.propTypes = {
+  /**
+   * Takes in a react-router `Link` reference and sets it
+   * as the base element. You may ONLY use it like the
+   * following:
+   * ```
+   * import { Link } from 'react-router'
+   * Link={Link}
+   * ```
+   */
+  Link: PropTypes.func,
+  /** Controls color of button */
+  bgColor: PropTypes.oneOf(VALID_BG_COLORS),
+  /** Controls the size of the button */
+  size: PropTypes.oneOf(['s', 'm']),
```

Comment:
Should we add support for a size smaller than `'s'` too? @akdetrick and I were talking about potentially having this replace the initial avatar in `MemberEdit` dropdown options: https://github.com/cbinsights/cbi-tables/blob/master/src/edit/MemberEdit/Img.jsx

Reply:
+1 on supporting `xs` here

Reply:
<img width="257" alt="Screen Shot 2019-12-10 at 5 04 14 PM" src="https://user-images.githubusercontent.com/52427513/70573374-d493d780-1b6f-11ea-9d7b-547d7473571f.png">

Asaf is hoping we can use the medium size ones here.